### PR TITLE
Process workflows stream

### PR DESCRIPTION
**Pull request by ypc-faros**
## Description

Processes the workflows GitHub stream

## Type of change
- [X] New feature

### CODE REVIEW

**Review by tovbinm — APPROVED** [review]
awesome!

please bump the version

**Comment by ypc-faros** [comment]
> awesome!
> 
> please bump the version

Yay 🥳 !

File: destinations/airbyte-faros-destination/src/converters/github/workflows.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    if (!repositoryKey) return [];
+
+    return [
+      {
+        model: 'cicd_Pipeline',
+        record: {
+          uid: workflow.id,
+          name: workflow.name,
+          url: workflow.url,
+          organization: repositoryKey.organization,
```
Comment (thomas-gerber) [inline destinations/airbyte-faros-destination/src/converters/github/workflows.ts:31]: @ypc-faros I think a cicd_Organization needs to be created as well somewhere (from this stream or another one?)
For example: 
* https://github.com/faros-ai/airbyte-connectors/blob/main/destinations/airbyte-faros-destination/src/converters/buildkite/organizations.ts
* https://github.com/faros-ai/airbyte-connectors/blob/main/destinations/airbyte-faros-destination/src/converters/jenkins/builds.ts#L30
@willmarks can you confirm?

Comment (tovbinm) [reply destinations/airbyte-faros-destination/src/converters/github/workflows.ts:31]: Ahh, good point. Adding...

Comment (ypc-faros) [reply destinations/airbyte-faros-destination/src/converters/github/workflows.ts:31]: Thanks @tovbinm !

**Pull request merged**